gradle.properties

```diff
@@ -17,13 +17,8 @@ versionName=0.0.1
 # Increase memory for in-process compiler execution.
 org.gradle.jvmargs=-Xmx3g
 
-# Because of: Please try to disable compiler caches and rerun the build. To disable compiler caches, add the following line to the gradle.properties file in the project's root directory
-kotlin.native.cacheKind=none
-
```

Comment:
Disabling cache for kotlin/native was used before to fix build issues but it can be safely removed now